src/Resources/config/services.xml

```diff
@@ -97,5 +97,15 @@
             <argument type="service"
                       id="Adyen\Shopware\Framework\Cookie\AdyenCookieProvider.inner"/>
         </service>
+
+        <!--gesagt.getan. services-->
```

Comment:
I would move this service to "Adyen Services" on line 19 instead

Reply:
moved it up